sharding/nodesCoordinator/hashValidatorShuffler.go

```diff
@@ -92,11 +92,11 @@ func NewHashValidatorsShuffler(args *NodesShufflerArgs) (*randHashShuffler, erro
 
 	log.Debug("Shuffler created", "shuffleBetweenShards", args.ShuffleBetweenShards)
 	rxs := &randHashShuffler{
-		shuffleBetweenShards:                     args.ShuffleBetweenShards,
-		availableNodesConfigs:                    configs,
-		enableEpochsHandler:                      args.EnableEpochsHandler,
-		stakingV4DistributeAuctionToWaitingEpoch: args.EnableEpochs.StakingV4DistributeAuctionToWaitingEpoch,
-		stakingV4EnableEpoch:                     args.EnableEpochs.StakingV4EnableEpoch,
+		shuffleBetweenShards:      args.ShuffleBetweenShards,
+		availableNodesConfigs:     configs,
+		enableEpochsHandler:       args.EnableEpochsHandler,
+		stakingV4Step3EnableEpoch: args.EnableEpochs.StakingV4Step3EnableEpoch,
+		stakingV4Step2EnableEpoch: args.EnableEpochs.StakingV4Step2EnableEpoch,
```

Comment:
for code consistency, switch L98 with L99?

Reply:
Right, changed as suggested